templates/Includes/CMSPagesController_ContentToolbar.ss

```diff
@@ -1,8 +1,8 @@
 <div class="cms-content-batchactions">
-	<div class="view-mode-batchactions-wrapper">
+	<label class="view-mode-batchactions-wrapper">
 		<input id="view-mode-batchactions" name="view-mode-batchactions" type="checkbox" />
-		<label for="view-mode-batchactions"><% _t("CMSPagesController_ContentToolbar_ss.MULTISELECT","Multi-selection") %></label>
-	</div>
+		<span><% _t("CMSPagesController_ContentToolbar_ss.MULTISELECT","Multi-selection") %></span>
```

Comment:
see https://github.com/silverstripe/silverstripe-framework/pull/4293#discussion-diff-32611791

tl;dr Add a class to the span (`view-mode-batchactions-label` perhaps?)